src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,73 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import { Livechat } from '../../../api';
+import I18n from '../../../i18n';
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+
+const DisplayCallIframe = (rid) => (
```

Comment:
Do we still need this component? I believe this isn't required now since we're opening jitsi on a new tab right?

Reply:
For our webrtc meet app, we need to display it in an iframe, so we will be needing this component.

Reply:
Okay makes sense. Lets keep it